network-api/networkapi/wagtailpages/constants.py

```diff
@@ -0,0 +1,2 @@
+# Validates whether a string is either a valid URL, a query string (?param=test), or both.
+url_or_query_regex = r"^(https?://[\w.-]+(/\S*)?)?(\?[\w-]+(=[\w-]*)?(&[\w-]+(=[\w-]*)?)*)?$"
```

Comment:
I moved this to a constants file because we are also using the same regex expression for the same type of validation [here](https://github.com/MozillaFoundation/foundation.mozilla.org/issues/10514). I am also opening this issue so we can discuss if this is the best place to store the regex moving forward, as I am assuming more and more fields are going to want the option to link to relative URL's such as `?form=donate`!